src/MollieApiClient.php

```diff
@@ -120,7 +120,7 @@ class MollieApiClient implements Connector
     /**
      * Version of our client.
      */
-    public const CLIENT_VERSION = '3.3.2';
+    public const CLIENT_VERSION = '3.3.3';
```

Comment:
New feature introduced, so:

```suggestion
    public const CLIENT_VERSION = '3.4.0';
```